tests/config/state/tests.yml

```diff
@@ -1,83 +1,106 @@
-# Supported operations: set, get, delete, bulkget, bulkset, bulkdelete, transaction, etag, first-write, query, ttl
+# Supported operations: transaction, etag, first-write, query, ttl
```

Comment:
set, get, delete, bulkget, bulkset, bulkdelete have been removed because they're always enabled.

set, get, and delete are required in every state store (otherwise, it wouldn't be a valid state store).
bulk* are included automatically for all components, even when they don't implement them natively.

Removing these operations allowed for simplifying the YAML below. Importantly, some operations were missing even though they should have been present (see how memcached had bulkset and bulkdelete but not bulkget).

Reply:
I agree but this schema needs to be updated. We should rename "operations" to "optionalOperations"

Reply:
After looking into the README file we have for conf tests, I think we should either refactor all conf tests to have this new behavior or keep it as is. Having it half-way is not a good contributor experience. Also, it is not clear if there will be anyone volunteering to completing this refactoring for all components.

Long story short, I am trying to minimize half-finished work in our master branch - at least when there is no clear commitment of bringing it to the finish line any time soon.

Reply:
Let's do this then. I will merge this and for 1.12 we change the meaning of operations to only be optional operations and required operations are always included in the test suite.